pulse/loaders/confluence.py

```diff
@@ -0,0 +1,43 @@
+"""Contains confluence loaders for loading data from confluence into the vectorstore."""
+import logging
+import sys
+
+from langchain.schema.vectorstore import VectorStore
+from lxml import etree
+from pulse.services.confluence import confluencefactory
+from pulse import config
+
+logging.basicConfig(level=logging.DEBUG, stream=sys.stdout)
+logger = logging.getLogger("pulse.loaders.confluence")
+
+def load_all_labelled_docs(vectorstore: VectorStore):
+    """
+    Load all documents from a confluence space with "pulse" label into the vector store
+    """
+    # rest call to confluence with the username and api key
+    docs = confluencefactory.client.get_all_pages_from_space(config.confluence_space, expand="body.view,version")
+    for doc in docs:
+        doc_loader(doc, vectorstore)
+        
+    return docs
+
+def doc_loader(doc, vectorstore: VectorStore):
```

Comment:
Return type!

Reply:
tbh don't know what it is & don't really need to return anything sooooo